src/Amadevus.RecordGenerator.Generators/SyntaxExtensions.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
```

Comment:
Not sure why this appeared here… No other changes in this file?

Reply:
Removed and sorted usings in all projects